docs/developer-guide/etendo-classic/concepts/Processes.md

```diff
@@ -0,0 +1,292 @@
+---
+tags: 
+  - PL/SQL Processes
+  - Java Processes
+  - Execution Methods
+  - Process Scheduling
+  - Parameters
+---
+#  Processes
+
+
+  
+##  Overview
+
+A process is a systematic series of actions directed to some end.  A process receives some parameters and taking them into account performs some actions to obtain a result. Etendo defines two main kinds of processes
+_PL/SQL Processes_ and _Java Processes_ .
+
+All processes (as well as [Reports](../../../developer-guide/etendo-classic/how-to-guides/How_to_create_a_Report.md)) are managed from the same window: `Application Dictionary > Report and Process`. 
+
+Once a process is defined it can be added to the menu to be called directly from there, it can be invoked through a button or it can be scheduled to be executed in the background.
+
+Background processes can be set as _Prevent Concurrent Executions_ . Before a process with this attribute is about to be executed, it is checked there are no other instances of the same process in execution at the same time, in this case another execution is aborted.
+
+###  Parameters
+
+When a process (PL/SQL or Java) or a Jasper Report is set as _Standard_, a pop-up message is automatically generated and displayed when invoking it, this message looks like this:
+
+![](../../../assets/developer-guide/etendo-classic/concepts/Processes-0.png)
+
+This pop-up has a section with information about the process, this information is obtained from the _Help_ field in the _Report and Process_ tab.
+
+It also prompts for a number of parameters, and finally there are two buttons, _OK_ to start the process and _Cancel_ to close the pop-up and not to do anything else. In case the process did not require any parameter that section would be empty.
+
+When this kind of process requires parameters they must be defined in the `Application Dictionary > Report and Process > Parameter`tab.
+
+
+![](../../../assets/developer-guide/etendo-classic/concepts/Processes-1.png)
+
+Parameters are defined in a very similar way than columns are. Depending on the _Reference_ field, the generated UI assigned for the parameter will be different.
+
+Let's overview some of the most important fields in this tab:
+
+  * *DB Column name* : This is the name the parameter will be passed with to the process. When the process is going to use this parameter, it will have to use this name. 
```

Comment:
```suggestion
  * _DB Column name_ : This is the name the parameter will be passed with to the process. When the process is going to use this parameter, it will have to use this name. 
```